fts/tests.py

```diff
@@ -449,9 +449,11 @@ def mockflatten(input_name, output_name, *args, **kwargs):
     body_text = browser.find_element_by_tag_name('body').text
     if len(warning_texts) == 0:
         assert 'Conversion Warnings' not in body_text
+        assert 'conversion errors' not in body_text
     else:
         assert warning_texts[0] in body_text
         assert 'Conversion Warnings' in body_text
+        assert 'conversion errors' not in body_text
```

Comment:
Conversion Warnings is the exact text used as the heading in the Conversion Warnings panel (L110). 
conversion errors is the exact text we have replaced.

A better test to see that 'conversion warnings' and 'Conversion Warnings' are both shown could be used, but I figured that what we have now would be good enough.